openupgrade_scripts/scripts/account/14.0.1.1/post-migration.py

```diff
@@ -605,6 +605,45 @@ def fill_account_journal_payment_credit_debit_account_id(env):
         )
 
 
+def create_new_counterpar_account_payment_transfer(env):
```

Comment:
Typo: create_new_counterpart_account_payment_transfer

Reply:
@royleviindoo could you adjust this?

Reply:
@LoisRForgeFlow ok, done